tpgtools/main.go

```diff
@@ -384,6 +416,49 @@ func generateSweeperFile(res *Resource) {
 	}
 }
 
+func generateResourceTestFile(res *Resource) {
+	if len(res.TestSamples()) < 1 {
+		return
+	}
+	// Generate resource file
+	tmplInput := ResourceInput{
+		Resource: *res,
+	}
+
+	tmpl, err := template.New("test_file.go.tmpl").Funcs(TemplateFunctions).ParseFiles(
+		"templates/test_file.go.tmpl",
+	)
+	if err != nil {
+		glog.Exit(err)
+	}
+
+	contents := bytes.Buffer{}
+	if err = tmpl.ExecuteTemplate(&contents, "test_file.go.tmpl", tmplInput); err != nil {
+		fmt.Println(contents.String())
+		glog.Exit(err)
+	}
+
+	if err != nil {
+		glog.Exit(err)
+	}
+
+	formatted, err := formatSource(&contents)
+	if err != nil {
+		err := ioutil.WriteFile("./temp/temp.go", []byte(contents.String()), 0644)
```

Comment:
Doesn't this overwrite the original format error?

Reply:
yeah this was debugging code... I've removed it